lib/engine/game/base.rb

```diff
@@ -775,6 +779,105 @@ def check_overlap(routes)
         end
       end
 
+      def check_connected(route, token)
```

Comment:
did you change anything here that i need to look at or is it just movement?

Reply:
Just moving game logic out of route